### DISCUSSION TITLE
Show miss count on score list view

### DISCUSSION THREAD

**Discussion by Dimmitsaras** [ppy/osu#30173] [Ideas]
![image](https://github.com/user-attachments/assets/591ebf90-0854-4f4e-9769-7d0bfd4ac42e)

Maybe pp potentially too?

**Comment by bdach** [comment]
Full hit statistics are already shown on hover though?

**Reply by Dimmitsaras** [reply]
It does, but given the recent pp changes, the number of misses has grown in importance. It would help find higher pp plays on the list at a glance, without having to hover each play individually

If the system is ranking plays based on a miss count metric, it should be easily assessable by the player when viewing the leaderboard.

**Reply by WitherFlower** [reply]
I think at this point your use case is covered by the "add pp leaderboard" proposal (https://github.com/ppy/osu/discussions/13024)
